DSCResources/MSFT_xSQLServerSetup/MSFT_xSQLServerSetup.psm1

```diff
@@ -43,10 +43,10 @@ function Get-TargetResource
         $PID,
 
         [System.String]
-        $UpdateEnabled = "True",
```

Comment:
I have described the issue in #16 with regarding current defaults.
In addition 2008 R2 setup doesn't support these, so by just removing defaults they no longer passed in as command parameters, this way you can use this resource for 2008 installation.

Other option would be to check version and remove $UpdateEnabled and $UpdateSource parameter array before running setup.

SQL Server setup has default value of $true anyway, so by removing this in Resource effectively doesn't change the behaviour as setup default takes affect.

https://msdn.microsoft.com/en-us/library/ms144259(v=sql.110).aspx